lib/edge/targeting.test.js

```diff
@@ -1,4 +1,4 @@
-import { PrebidORTB2, TargetingKeyValues } from "./targeting";
+import { LoblawMediaPrivateID, PrebidORTB2, TargetingKeyValues } from "./targeting";
```

Comment:
You don't seem to use the newly added import. Added by mistake?

Reply:
Fixed